app/components/settings/LocationInput.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useState } from "react";
+import styles from "./Settings.module.css";
+
+interface Location {
+  latitude: number;
+  longitude: number;
+}
+
+interface Props {
+  value: Location | undefined;
+  onChange(value: Location | undefined): void;
+}
+
+const defaultLocation = fetch("http://ip-api.com/json")
```

Comment:
this works locally, but chrome doesn't like cross origin non-ssl requests, is there a way to get it through the browser?

Reply:
switched to an https api